CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing Pull Requests
+
+## What to work on
+
+We welcome small pull request contributions from anyone (docs improvements, bug fixes, minor features.) as long as they follow a few guidelines:
+
+- For very minor changes like correcting a typo feel free to just send a pull request without any ceremony. Otherwise ... 
+- Please start by [choosing an existing issue](https://github.com/radius-project/samples/issues), or [opening an issue](https://github.com/radius-project/samples/issues/new/choose) to work on.
+- The maintainers will respond to your issue, please work with the maintainers to ensure that what you're doing is in scope for the project before writing any code.
+- If you have any doubt whether a contribution would be valuable, feel free to ask.
+
+We the maintainers have discretion over what features and pull requests we accept. Please understand that we are responsible for the long-term support and maintenance of Radius, and so we sometimes need to make hard decisions to limit the scope. For another perspective on this, we really like this [article](https://www.igvita.com/2011/12/19/dont-push-your-pull-requests/).
+
+## Sending a pull request
+Please submit pull requests using a forked repo and open pull requests against the `edge` branch (the default) unless otherwise instructed.
```

Comment:
```suggestion

Please submit pull requests using a forked repo and open pull requests against:

- The default, versioned branch (`v0.30` for example) if you are adding/fixing a sample and it is compatible with the latest release of Radius.
- The `edge` branch if you are contributing a sample that requires a feature that will be available in the next release of Radius.
```

Hmmm it's a little nuanced based on what you're trying to do. Lemme see if I can take a crack at it above

Reply:
Makes sense, thank you! :)